setup.py

```diff
@@ -0,0 +1,13 @@
+from setuptools import find_packages, setup
+
+setup(
+    name="phcode42v2",
+    version="1.0.0",
+    description="Code42 for Phantom",
+    packages=find_packages(),
+    install_requires=["requests>=2.3",
+                      "pytest==4.4.0",
+                      "pytest-mock==1.10.3",
+                      "py42>=1.14.1",
+                      "phantomstubs"]
```

Comment:
We could actually move all of `phantomstubs` to just be in a `phantom` folder under the main project, and just have the compile logic skip that dir.

Reply:
Something like that might look a little odd and the surface and what we're doing should probably be called out in the readme

Reply:
I think it's a bit more readable to have the top level directory for the stubs read `phantomstubs` rather than just `phantom`. The `phantomstubs` dir is already excluded from the archive compilation in `exclude_files.txt` -- I'm kind of confused by what you are suggesting here?

Reply:
I like what @alanag13 is suggesting. At minimum, it would be nice to have the package name be just `phantom` so the imports work

Reply:
Implemented Alan's suggestion